python/cudf_polars/cudf_polars/experimental/io.py

```diff
@@ -283,8 +286,11 @@ def _(
         paths = list(ir.paths)
         if plan.flavor == ScanPartitionFlavor.SPLIT_FILES:
             # Disable chunked reader when splitting files
-            config_options = ir.config_options.set(
-                name="parquet_options.chunked", value=False
+            config_options = dataclasses.replace(
```

Comment:
`dataclasses.replace` doesn't handle "nested" replacements like `config_options.parquet_options.chunked`. This was the only place using `ConfigOptions.set` so I've just done it manually. If this becomes a pain we can define a helper to do it dynamically.

Reply:
I'm hoping this pattern isn't common.
It may make sense to just disable the chunked parquet reader for the `"streaming"` executor in all cases.